autonomi/src/lib.rs

```diff
@@ -85,6 +88,14 @@ pub use libp2p::Multiaddr;
 
 #[doc(inline)]
 pub use client::{
+    // Client Configs
+    config::ChunkOperationConfig,
+    config::ClientConfig,
+    config::ClientOperationConfig,
+    config::GraphOperationConfig,
+    config::PointerOperationConfig,
+    config::ScratchpadOperationConfig,
```

Comment:
Making this one type will make the exposed types much simpler, 6 exposed public structs related to config is huge